pycbc/detector/ground.py

```diff
@@ -474,6 +474,45 @@ def time_delay_from_detector(self, other_detector, right_ascension,
                                              right_ascension,
                                              declination,
                                              t_gps)
+    
+    def convert_tc(self, ref_tc, ra, dec, ref_frame='geocentric'):
```

Comment:
@acorreia61201 Is is there a clearer name for this? It seems like it is always giving you the time *at* the current detector. Is that not right? 'convert_tc' seems a bit vague. 

This seems more like a function should be called something like 'arrival_time'  (e.g. det.arrival_time(...)) or something similarly description so the function is somewhat clear in the name.